src/components/Explore/TokenDetails/TokenDetail.tsx

```diff
@@ -198,6 +201,16 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
   const chainInfo = getChainInfo(token?.chainId)
   const networkLabel = chainInfo?.label
   const networkBadgebackgroundColor = chainInfo?.backgroundColor
+  const [tokenPrice, setTokenPrice] = useState(0)
+  const [tokenDelta, setTokenDelta] = useState<number | undefined>()
+
+  const setTokenNumbers = useCallback(
```

Comment:
you can use `jotai` atoms so you don't need to pass this callback function into the other component